### PR TITLE
LPD-38811 avoid duplicate ERC exception

### PR DESCRIPTION
**Pull request by andrea-ale-sbarra**
https://liferay.atlassian.net/browse/LPD-38811

### CODE REVIEW

**Comment by andrea-ale-sbarra** [comment]
ci:test:sf

File: modules/apps/commerce/commerce-product-service/src/main/java/com/liferay/commerce/product/service/impl/CPDefinitionLocalServiceImpl.java [hunk 1]
```diff
@@ -1138,11 +1138,16 @@ public CPDefinition copyCPDefinition(
 					(CPDefinitionSpecificationOptionValue)
 						cpDefinitionSpecificationOptionValue.clone();
 
+			long cpDefinitionSpecificationOptionValueId =
+				counterLocalService.increment();
```
Comment (alessiorendina) [inline modules/apps/commerce/commerce-product-service/src/main/java/com/liferay/commerce/product/service/impl/CPDefinitionLocalServiceImpl.java:1142]: Can you move it down where it is used?

**Review by alessiorendina — CHANGES_REQUESTED** [review]
@andrea-ale-sbarra LGTM overall, just one small change request. Thank you!

**Comment by andrea-ale-sbarra** [comment]
ci:test:relevant

**Comment by andrea-ale-sbarra** [comment]
ci:test:sf

**Comment by andrea-ale-sbarra** [comment]
ci:test:relevant

**Comment by andrea-ale-sbarra** [comment]
ci:test:sf

**Comment by andrea-ale-sbarra** [comment]
ci:forward:force

**Pull request closed without merging**